stories/dynamic-page/dynamic-page.stories.js

```diff
@@ -0,0 +1,929 @@
+                </div>
+                <div class="fd-bar__element">
+                    <button aria-label="button" class="fd-button fd-button--transparent fd-button--compact">Cancel</button>
+                </div>
+            </div>
+        </div>
+    </div>
+
+    <br /><br />
+
+    <div class="fd-dynamic-page" role="region" aria-label="Dynamic Page Mobile Summary Line Example">
+        <div class="fd-dynamic-page__summarized-title-area fd-dynamic-page__summarized-title-area--s fd-dynamic-page__summarized-title-area--no-shadow" tabindex="0">
+            <div class="fd-dynamic-page__summarized-title fd-dynamic-page__summarized-title--collapsed"> Balenciaga Tripple S Trainers </div>
+            <div class="fd-dynamic-page__toolbar fd-dynamic-page__toolbar--actions fd-toolbar fd-toolbar--clear fd-toolbar--transparent">
+                    <button class="fd-button fd-button--transparent" aria-label="Close">
+                        <i class="sap-icon--slim-arrow-down"></i>
+                    </button>
+            </div>
+        </div>
+        <div class="fd-dynamic-page__title-area fd-dynamic-page__title-area--s fd-dynamic-page__title-area--collapsed" aria-hidden="true" tabindex="0">
+            <div class="fd-dynamic-page__breadcrumb-container">
+            <ul class="fd-dynamic-page__breadcrumb--s fd-breadcrumb">
+                <li class="fd-breadcrumb__item"><a class="fd-breadcrumb__link" tabindex="0" href="#">Men</a></li>
+                <li class="fd-breadcrumb__item"><a class="fd-breadcrumb__link" tabindex="0" href="#">Shoes</a></li>
+                <li class="fd-breadcrumb__item"><a class="fd-breadcrumb__link" tabindex="0" href="#">Running Shoes</a></li>
+            </ul>
+            <div class="fd-dynamic-page__toolbar fd-dynamic-page__toolbar--actions fd-toolbar fd-toolbar--clear fd-toolbar--transparent">
+                    <button class="fd-button fd-button--transparent" aria-label="Close">
+                        <i class="sap-icon--decline"></i>
+                    </button>
+            </div>
+            </div>
+            <div class="fd-dynamic-page__title-container">
+                <div class="fd-dynamic-page__title fd-dynamic-page__title--collapsed"> Balenciaga Tripple S Trainers </div>
+                    <div class="fd-dynamic-page__toolbar fd-dynamic-page__toolbar--actions fd-toolbar fd-toolbar--clear fd-toolbar--transparent">
+                        <button class="fd-button fd-button--transparent" aria-label="More Header Content">
+                            <i class="sap-icon--overflow"></i>
+                        </button>
+                        <button class="fd-button fd-button--transparent" aria-label="Header actions">
+                            <i class="sap-icon--overflow"></i>
+                        </button>
+                    </div>
+            </div>
+            <div class="fd-dynamic-page__subtitle"> Collapsed title area and header with Summary Line </div>
```

Comment:
"Summary line" shouldn't be capitalized ever, and anything else really. Only sometimes when components are referred to in documentation that are linked to other pages. In this case, it would've been the mention of **Dialog** in the main description.

Unless of course it's the first letter of the first word in a sentence.

Reply:
modified accordingly